modules/ROOT/pages/logging-mdc.adoc

```diff
@@ -0,0 +1,112 @@
+= MDC Logging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: tracing, correlation, id, logging, mdc, variables, anypoint, studio
+
+The idea behind MDC (_Mapped Diagnostic Context_) is to enrich logging to provide more context or information that could be available in the current event, to improve tracking.
+
+In the following example without MDC:
+
+----
+<flow name="exmapleFlow">
+    <http:listener config-ref="httpConfig" path="/modifyLoggingContext" />
+    <logger message="Example"/>
+</flow>
+----
+
+A usual logging output is:
+
+----
+INFO  2021-04-08 16:58:26,869 [[MuleRuntime].uber.15: [test-project-app].exmapleFlow.CPU_LITE @18f679] [processor: exmapleFlow/processors/1; event: b5a95a63.f190.4c91.880a.5b54512fa6b1] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: Example
+----
+
+With the Mule Tracing Module, you can add, remove and clear variables from the Logging Context for a given Mule Event, to enhance your logs. This context will endure for the entire execution of said Event.
+
+Consider the following example:
+----
+<flow name="exmapleFlow">
+    <http:listener config-ref="httpConfig" path="/modifyLoggingContext" />
+    <tracing:set-logging-variable variableName="testVar" value="Hello World!" />
+    <logger message="Example"/>
+</flow>
+----
+
+After executing the flow, the output logs are:
+----
+INFO  2021-04-08 16:58:26,882 [[MuleRuntime].uber.15: [test-project-app].exmapleFlow.CPU_LITE @18f679] [{correlationId=c85e16c0-98a4-11eb-bc34-cac765a2219b, processorPath=exmapleFlow/processors/2, testVar=Hello World!}] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: Example
+----
+
+This logging context will affect any output of a Logger Message Processor, as well as any internal logging produced by the Mule Runtime. Therefore, you could add more context to the event that is being processed in case of needing additional information for tracking down any potential issue (e.g. an unexpected error occurred).
+
+==== Installing and Setting up the Mule Tracking Module
+
+To use the MDC Logging operations, you need to add the Mule Tracing Module to your application.
+
+// Add explanation how to add the module as any other connector
+
+You also have to modify the `log4j.xml` of your application or the entire runtime (in `conf/log4j2.xml`). You will have to modify the pattern layouts from:
```

Comment:
We have to define if it's needed in the docs or it will be the default

Reply:
For the moment, I agree with adding it. If we decide to change the default I would move it to some specific section to "customize" which entries are going to be shown